packages/plugin-compiler/src/test/resources/sample/input/Sample.kt

```diff
@@ -82,6 +84,33 @@ class Sample : RealmObject {
     var nullableTimestampListField: RealmList<RealmInstant?> = realmListOf()
     var nullableObjectIdListField: RealmList<ObjectId?> = realmListOf()
 
+    // Set types
```

Comment:
Do we support `Set<List<T>>`? I don't remember 🙈 If we don't, maybe add a comment about it as well.

Reply:
Thanks for highlighting this. I just realised I'm missing compiler tests for sets too. I will add them and test it there.

Reply:
Oh, or do you mean whether Core itself supports that... I'll ask it but I doubt it.

Reply:
Apparently not.